org.eclipse.scanning.test/src/org/eclipse/scanning/test/validation/ScanRequestValidationTest.java

```diff
@@ -65,7 +65,7 @@ public void legalDetectorModelList() throws Exception {
 		
 		GridModel gmodel = new GridModel("stage_x", "stage_y");
 		gmodel.setBoundingBox(new BoundingBox(10, -10, 100, -100));
-		ScanRequest<?> req = new ScanRequest<>(gmodel, null);
+		ScanRequest<?> req = new ScanRequest<>(gmodel, null, null, null);
 		req.putDetector("mandelbrot", new MandelbrotModel());

```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "mandelbrot" 13 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)